src/current/_includes/metrics-usage/tenant.consumption.write_batches.md

```diff
@@ -0,0 +1,5 @@
+The number of RUs that were consumed due to KV writes, broken down by batches. SQL statements are translated into lower-level KV write requests that are sent in batches. Batches may contain any number of requests. Requests can have a payload containing any number of bytes. Write operations are replicated to multiple storage processes (3 by default), with each replica counted as a separate write operation.Storage layer I/O is converted to Request Units using this equivalency:
```

Comment:
typo: no space between "operation.Storage layer"

strongly suggest removing "(3 by default)" and linking to 

https://www.cockroachlabs.com/docs/v23.2/architecture/storage-layer#compaction 

because that is already documented there using an include.  Further, it should not be changed casually (see linked doc)

Reply:
I fixed the typo.

I left `(3 by default)` because this is important for Serverless and added a link on `storage processes` to https://www.cockroachlabs.com/docs/cockroachcloud/serverless-resource-usage#understand-resource-consumption-in-cockroachdb-serverless